scripts/seed.ts

```diff
@@ -0,0 +1,114 @@
+import { config } from 'dotenv';
+import pg from 'pg';
+
+config();
+
+async function seed() {
+  const client = new pg.Client({
+    user: process.env.POSTGRES_USER,
+    password: process.env.POSTGRES_PASSWORD,
+    host: process.env.POSTGRES_HOST,
+    port: parseInt(process.env.POSTGRES_PORT || '5432'),
+    database: process.env.POSTGRES_DB,
+    ssl: process.env.POSTGRES_SSL === '1',
+  });
+
+  try {
+    await client.connect();
+
+    // Create admin business entity
+    const adminEntityResult = await client.query(`
+      INSERT INTO accounter_schema.financial_entities (type, name)
+      VALUES ('business', 'Admin Business')
+      RETURNING id
+    `);
+
+    console.log('✅ Created admin business entity');
+
+    const adminEntityId = adminEntityResult.rows[0].id;
+
+    // Create corresponding business record
+    await client.query(
+      `
+      INSERT INTO accounter_schema.businesses (id)
+      VALUES ($1)
+    `,
+      [adminEntityId],
+    );
+
+    console.log('✅ Created corresponding business record');
+
+    // update the owner_id of the admin entity
+    await client.query(
+      `
+      UPDATE accounter_schema.financial_entities 
+      SET owner_id = $1
+      WHERE id = $1
+    `,
+      [adminEntityId],
+    );
+
+    console.log('✅ Updated admin business entity owner_id to itself');
```

Comment:
I'd suggest less technical message, maybe `business set as admin`. what do you think?

Reply:
This file was mostly AI generated. But I can adjust the messages. It's focused for devs working on the platform btw.